files/ko/web/javascript/reference/global_objects/dataview/setuint8/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: DataView.prototype.setUint8()
+slug: Web/JavaScript/Reference/Global_Objects/DataView/setUint8
+l10n:
+  sourceCommit: e01fd6206ce2fad2fe09a485bb2d3ceda53a62de
+---
+
+{{JSRef}}
+
+{{jsxref("DataView")}} 인스턴스의 **`setUint32()`** 메서드는 숫자를 받아
+이 `DataView`의 지정된 바이트 오프셋의 1 바이트의 공간에 8비트 부호 없는 정수로 저장합니다.
```

Comment:
원문과의 diff를 더 명확하게 알기위해 쪼개지 않는건 어떤가요? line으로 바로 찾을수 있는 장점이 있을것 같습니다~!

혹시 이전에 논의가 끝난 내용이라면 린트 룰이 있으면 좋을것 같네요.

Reply:
음.... 한글과 영어의 80 기준을 그대로 적용하자면 한국어와 영어의 구조가 달라 나름대로 문제가 될것 같습니다. 예를 들어 [Introduction to the DOM](https://developer.mozilla.org/en-US/docs/Web/API/Document_Object_Model/Introduction) 을 예로 들어보면 

영어본문
```
The **Document Object Model** (_DOM_) is the data representation of the objects
that comprise the structure and content of a document on the web.
This guide will introduce the DOM,
look at how the DOM represents an {{Glossary("HTML")}} document in memory
and how to use APIs to create web content and applications.
```

한글 문서
```
**도큐먼트 오브젝트 모델**(_DOM_)은 웹에서 문서의 구조와 내용을 구성하는 객체를 데이터로 표현한 것입니다.
이 가이드에서는 DOM을 소개하고, DOM이 메모리에서 {{Glossary("HTML")}} 문서를 어떻게 나타내는지,
그리고 API를 사용하여 웹 콘텐츠와 애플리케이션을 만드는 방법을 살펴봅니다.
```

인데 Line to Line 으로 1:1 매칭으로 하기도 애매하고... 영어보다 보통 짧은 한국어 문장 특성상 2줄이 1줄 80자 내로 들어오는 일도 비일비재해서요. 가이드만 정해지면 따라도록 하겠습니다.

Reply:
> 정확하게 기억은 안나지만 80자 이하 권장 정책을 봐서 가능하면 쪼개고 있긴 합니다. 

@wisedog 님, "80자 이하 권장 정책" 관련하여 저도 보았던것같은데, 관련 내용을 못찾겠네요 ㅎㅎ..

> 원문과의 diff를 더 명확하게 알기위해 쪼개지 않는건 어떤가요? line으로 바로 찾을수 있는 장점이 있을것 같습니다~!

Markdown은 간결하고 읽기 쉬운 문서를 만들기 위한 설계 의도 중 하나로 기본적으로 개행이 공백으로 처리가 됩니다. 따라서, 마크다운 
 설계 의도에 맞게 가독성이 좋아진다고 생각하여 개행에 대해 별다른 의견을 남기지 않고 병합을 해왔습니다.

그동안은 제목 단위로 영역을 찾았기에 큰 불편함은 못 느꼈습니다.

@1ilsang 님 주신 의견도 굉장히 좋다고 생각합니다. @mdn/yari-content-ko, 위와 같은 의견을 주셨는데 어떻게 생각하시나요!

Reply:
`en-US` 에서 점점 80자 룰이 적용된 문서가 많아지고 있어서 동일하게 1줄로 한다고 해도 나중에 `en-US` 에서 변경되면 그것도 안맞게 되는지라 80자 룰을 확인 후에 정하면 되리라 생각합니다. 일단 저도 제목 + 문단 단위로 찾고 있어서 큰 불편함을 느끼진 못하고 있습니다.

Reply:
@mdn/localization-team-leads @mdn/mdn-community-engagement Hello!

The conversation above addressed setting guidelines for the maximum number of words per line in Markdown files within the ko-locale. We would like to write a guide related to it.

**I'm keen to learn if there will be a guideline for the single-line character limit in Markdown files across the entire [content repo](https://github.com/mdn/content) (or translated-content repo).** (If possible) If such a guideline exists, could you specify the document where it's mentioned?

As shown below, in the content repo, documents have recently(..?) been written divided into multiple lines rather than single lines, so I asked if there was a related direction. I think there was a direction before that recommended writing in 80 characters or less, but I couldn't find it when I looked for it.

- https://github.com/mdn/content/blob/27b116a3801c7e718e9c68f578e89da2bd87fff2/files/en-us/web/javascript/reference/global_objects/symbol/keyfor/index.md?plain=1#L10

Reply:
Heya 👋🏻 - we can start a GitHub discussion to track this in case the conversation is lost in this PR, if you like.

As far as character limit goes, I am not aware of any guideline that says you should use max X chars per line.

>I think there was a direction before that recommended writing in 80 characters or less

You can definitely some existing docs where this was a preferred authoring style by some writers, but I have not seen any new documentation added at all that uses 80 char limit. In fact, if this was a requirement, it would probably be enforced with linting / code style tools.

We do have one guideline [about whitespace which says](https://developer.mozilla.org/en-US/docs/MDN/Community/Pull_requests#open_a_pull_request)

>Don't unnecessarily add or remove line breaks on pages that follow a certain formatting style.

So if a page uses 80 chars limit for a lot of text, don't make whitespace-only changes to change the style.

Reply:
@bsmth Thank you for your reply! I've created the discussion below for tracking.

@1ilsang , @wisedog It would be good to continue the conversation https://github.com/orgs/mdn/discussions/655.